contracts/script/deploy/router/output/out.json

```diff
@@ -0,0 +1,3 @@
+{
+  "eigenDACertVerifierRouter": "0xC8E6a718Ca497DeF385CB76e702b9A071961F90f"
```

Comment:
which network is this deployed on? Do we really need this checked in to the repo? If we do, then I'd recommend we follow the foundry best practices #7 in https://book.getfoundry.sh/guides/best-practices#scripts, do organize outputs by chain_id.

Reply:
0882b7e88b62b3e60f49163ecbdc292fc028a1db